pages/en/about/get-involved/events.mdx

```diff
@@ -3,22 +3,26 @@ title: Upcoming Events
 layout: about.hbs
 ---
 
-## Upcoming Node.js® Summits
+## Upcoming Events
 
-Interested in joining a [Collaborator Summit](/about/get-involved/collab-summit) hosted by the Node.js project?
-Check out the list below to find upcoming events.
+Node.js events are open and available to the public. Anyone is welcome to join and participate.
 
-Browse [previous Collaborator Summits & Events](/blog/events/) hosted by Node.js.
+[Collaborator Summits](/about/get-involved/collab-summit) hosted by the Node.js project are larger gatherings usually taking place adjacent to a community conference.
```

Comment:
retained this snippet about the collab summit in case someone finds this page first